README.md

```diff
@@ -662,21 +723,30 @@ HTTP/1.1 200 OK
 }
 ```
 
-#### Deleting Subscription
-To delete subscription one should run DELETE request on /subscriptions/{subscriptionId}
+#### Deleting a Subscription
+
+To delete a Subscription, send a DELETE request to the Subscription resource using its `id` field (`subscriptions/{id}`):
```

Comment:
@dehora @v-stepanov what is the progress here guys?, can we merge the branch and keep the PL open to follow up on these minor remarks?